plan/ROADMAP.md

```diff
@@ -0,0 +1,39 @@
+# Homestar Roadmap
+
+
+
+
+
+- Networking
+  - Receipt DHT(?)
+  - pubsub/gossipsub/episub
+- Blockstore
+  - Host
+    - Load Wasm & other data by CID
+    - "Content handles"
+  - Guest
+    - Block streaming API (in & out)
+- Standard Wasm library
+- (Co)effect plugin system
+  - First-party effects:
+    - HTTPS
+    - DNS
+    - Randomness oracle
+    - Encryption/decryption
+- Dynamic limits
+  - Fuel
+  - Memory
+  - Storage
+- Safety
+  - Dataflow checking
+- Trust
+  - UCAN (re)delegation support
+  - UCAN invocation support
```

Comment:
Just by looking at a few of these, some are in an in-progress or complete state, but we want to capture those as well generally yeah?

Reply:
Yup exactly 💯